_playwright-tests/UI/UploadRepo.spec.ts

```diff
@@ -22,6 +22,9 @@ test.describe('Upload Repositories', () => {
       const nameInput = page.getByPlaceholder('Enter name');
       await nameInput.click();
       await nameInput.fill(uploadRepoName);
+      // Upload can fail if repository is not valid HMS-9856
+      const newRepoRow = await getRowByNameOrUrl(page, uploadRepoName);
+      await expect(newRepoRow.getByText('Valid')).toBeVisible();
```

Comment:
this is before the repo is created so this will fail, we'd need to wait for the status to be valid after bulk_create is called. similar to where you added the check in the integration test